examples/hybrid_job.py

```diff
@@ -0,0 +1,48 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from braket.aws import AwsDevice
+from braket.circuits import Circuit, FreeParameter, Observable
+from braket.devices import Devices
+from braket.jobs import hybrid_job
+from braket.jobs.metrics import log_metric
+
+device_arn = Devices.Amazon.SV1
+
+
+@hybrid_job(device=device_arn, wait_until_complete=True)  # choose priority device
```

Comment:
nit: I think the comment is confusing now. Maybe even just using `Devices.Amazon.SV1` here directly instead of an intermediate variable would make it clearer